internal/impl/ollama/base_processor.go

```diff
@@ -86,6 +116,39 @@ var ollamaProcess = singleton.New(singleton.Config[*exec.Cmd]{
 		return cmd.Wait()
 	}})
 
+func downloadOllama(ctx context.Context, path string) error {
+	var url string
+	const baseURL string = "https://github.com/ollama/ollama/releases/download/v0.3.1/ollama"
+	switch runtime.GOOS {
```

Comment:
should this be configurable.

Reply:
with a default.

Reply:
Good point. This will be the default, and I'll add a string to support overriding.

Reply:
Done